closure/goog/events/eventid.js

```diff
@@ -18,15 +18,15 @@ goog.provide('goog.events.EventId');
 
 /**
  * A templated class that is used when registering for events. Typical usage:
- * <code>
- *   /** @type {goog.events.EventId<MyEventObj>}
+ * <pre>
```

Comment:
Unfortunately that seems to not work: then the `&gt;` does not show up as `<` in the docs.  See https://github.com/google/closure-library/pull/603#issuecomment-156789878